src/apps/eos/sign_tx.py

```diff
@@ -0,0 +1,61 @@
+from trezor import wire
+from trezor.crypto.curve import secp256k1
+from trezor.crypto.hashlib import sha256
+from trezor.messages.EosSignedTx import EosSignedTx
+from trezor.messages.EosSignTx import EosSignTx
+from trezor.messages.EosTxActionRequest import EosTxActionRequest
+from trezor.messages.MessageType import EosTxActionAck
+from trezor.utils import HashWriter
+
+from apps.common import paths
+from apps.eos import writers
+from apps.eos.actions import process_action
+from apps.eos.helpers import validate_full_path
+from apps.eos.layout import require_sign_tx
+
+
+async def sign_tx(ctx, msg: EosSignTx, keychain):
+    check(msg)
+
+    await paths.validate_path(ctx, validate_full_path, path=msg.address_n)
+
+    node = keychain.derive(msg.address_n)
+    sha = HashWriter(sha256())
+    await _init(ctx, sha, msg)
+    await _actions(ctx, sha, msg.num_actions)
+    writers.write_variant32(sha, 0)
+    writers.write_bytes(sha, bytearray(32))
+
+    digest = sha.get_digest()
+    signature = secp256k1.sign(
+        node.private_key(), digest, True, secp256k1.CANONICAL_SIG_EOS
+    )
+
+    resp = EosSignedTx()
```

Comment:
instead: `return EosSignedTx(signature)`

Reply:
Actually, after an internal discussion, we decided to stick with the unmodified protobuf
please revert the change back to r, s, v. Sorry about that.

Reply:
Reverted change and added return EosSignedTx with constructor initialization